packages/context/src/__tests__/unit/resolution-session.unit.ts

```diff
@@ -74,7 +74,7 @@ describe('ResolutionSession', () => {
     expect(session.currentInjection).to.be.exactly(injection);
     expect(session.injectionStack).to.eql([injection]);
     expect(session.getBindingPath()).to.eql('a --> b');
-    expect(session.getInjectionPath()).to.eql('MyController.constructor[0]');
+    expect(session.getInjectionPath()).to.eql('@MyController.constructor[0]');
```

Comment:
Let's not break the current injection path. Please revert the `@` prefix.

Reply:
Updated